tests/test_bitshuffle_leftovers.c

```diff
@@ -96,7 +96,7 @@ int main(void) {
    * (not supported by bitshuffle) and in addition, it is not
    * divisible by 8 (typesize) again.
    */
-  int size = 641092;
+  int size = 641096;
```

Comment:
With that, size *is* divisible by 8, making the comment above untrue. After thinking twice, this module is testing the situation where the chunksize is not a multiple of typesize, and we are not supporting this anymore.  So I'd rather keep the previous value (641092) and test that `blosc_compress()` is raising an error (like `mu_assert("ERROR: undetected chunksize not being a multiple of typesize", cbytes2 < 0)`.